core/ui/SideBar/NewTabButton.tid

```diff
@@ -0,0 +1,14 @@
+created: 20160319212903860
+modified: 20160319221316664
```

Comment:
It is difficult to get away with them. TiddlyWiki by default adds them if you create the tiddler through TiddlyWiki. You either have to manually delete the lines after the fact in your own text editor or do not construct content through TiddlyWiki. This disparity seems counter productive. Perhaps there could be a way to disable the auto time stamping or by default skip auto timestamping for tiddlers who's names begin with `$:/`